src/Go/boardAnalysis/goAI.ts

```diff
@@ -62,6 +62,7 @@ export function handleNextTurn(boardState: BoardState = Go.currentGame, useOffli
     // The game is over. We shouldn't get here in most circumstances,
     // because when the game ends resetAI() will be called to resolve promises.
     // Return an already-resolved promise until a new game is started.
+    resetAI(true);
```

Comment:
This shouldn't be needed (endGoGame *is* being properly called, here), but I suppose it doesn't hurt.